docs/source/reference/templates.md

```diff
@@ -22,16 +25,15 @@ Jinja provides a mechanism to [extend templates](http://jinja.pocoo.org/docs/2.1
 A base template can define a `block`, and child templates can replace or
 supplement the material in the block. The
 [JupyterHub templates](https://github.com/jupyterhub/jupyterhub/tree/HEAD/share/jupyterhub/templates)
-make extensive use of blocks, which allows you to customize parts of the
-interface easily.
+make extensive use of blocks, which allows you to customize parts of the interface easily.
 
 In general, a child template can extend a base template, `page.html`, by beginning with:
 
 ```html
 {% extends "page.html" %}
 ```
 
-This works, unless you are trying to extend the default template for the same
+This works if you are trying to extend the default template for the same
```

Comment:
This change inverts the logic making it inaccurate. It works _unless_ you do this. It _doesn't_ work _if_ you are doing it.

Reply:
okay, thanks.